pkg/conf/config.go

```diff
@@ -36,10 +36,10 @@ func EnvConfig(appName string, conf interface{}) error {
 }
 
 // GetBuilderKey returns the key to be used as token to interact with deis-controller
-func GetBuilderKey() (string, error) {
-	builderKeyBytes, err := ioutil.ReadFile(builderKeyLocation)
+func GetBuilderKey(builderKeyPath string) (string, error) {
```

Comment:
Just to be on the safeside that it we don't change it by mistake in other part of the code.I am sure that doesn't happen so i am ok with changing it to `var`